cypress/e2e/awx/infrastructure/instance-groups/instanceGroupsK8s.cy.ts

```diff
@@ -0,0 +1,267 @@
+      });
+  });
+
+  it('can visit the instances tab of an instance group and run a health check from row against an instance', () => {
+    cy.navigateTo('awx', 'instance-groups');
+    cy.verifyPageTitle('Instance Groups');
+    cy.filterTableBySingleSelect('name', instanceGroup.name);
+    cy.get('[data-cy="name-column-cell"]').click();
+    cy.url().then((currentUrl) => {
+      expect(currentUrl.includes('details')).to.be.true;
+      expect(currentUrl.includes('infrastructure/instance-groups')).to.be.true;
+    });
+    cy.clickTab(/^Instances$/, true);
+    cy.get('[data-ouia-component-id="simple-table"]').within(() => {
+      cy.get('tbody tr').should('have.length', 1);
+    });
+    cy.filterTableBySingleSelect('hostname', instance.hostname);
+    cy.intercept('POST', awxAPI`/instances/*/health_check/`).as('runHealthCheck');
+    cy.clickTableRowPinnedAction(instance.hostname, 'run-health-check', false);
+    cy.wait('@runHealthCheck')
+      .then((response) => {
+        expect(response.response?.statusCode).to.eql(200);
+      })
+      .its('response.body.msg')
+      .then((response) => {
+        expect(response).contains(`Health check is running for ${instance.hostname}.`);
+      });
+  });
+
+  it('can visit the details page of an instance nested inside an instance group and run health check on it', () => {
+    cy.navigateTo('awx', 'instance-groups');
+    cy.verifyPageTitle('Instance Groups');
+    cy.filterTableBySingleSelect('name', instanceGroup.name);
+    cy.get('[data-cy="name-column-cell"]').click();
+    cy.url().then((currentUrl) => {
+      expect(currentUrl.includes('details')).to.be.true;
+      expect(currentUrl.includes('infrastructure/instance-groups')).to.be.true;
+    });
+    cy.clickTab(/^Instances$/, true);
+    cy.get('[data-ouia-component-id="simple-table"]').within(() => {
+      cy.get('tbody tr').should('have.length', 1);
+    });
+    cy.filterTableBySingleSelect('hostname', instance.hostname);
+    cy.get('[data-cy="name-column-cell"]').click();
+    cy.url().then((currentUrl) => {
+      expect(currentUrl.includes('details')).to.be.true;
+      expect(currentUrl.includes('infrastructure/instance-groups')).to.be.true;
+    });
+    cy.verifyPageTitle(instance.hostname);
+    cy.contains('nav[aria-label="Breadcrumb"]', 'Instance groups').should('exist');
+    cy.contains('nav[aria-label="Breadcrumb"]', instanceGroup.name).should('exist');
+    cy.contains('nav[aria-label="Breadcrumb"]', 'Instances').should('exist');
+    cy.contains('nav[aria-label="Breadcrumb"]', instance.hostname).should('exist');
+    cy.contains('nav[aria-label="Breadcrumb"]', 'Details').should('exist');
+    cy.intercept('POST', awxAPI`/instances/${instance.id.toString()}/health_check/`).as(
+      'runHealthCheck'
+    );
+    cy.getByDataCy('run-health-check').click();
+    cy.wait('@runHealthCheck')
+      .then((response) => {
+        expect(response.response?.statusCode).to.be.true;
```

Comment:
```suggestion
        expect(response.response?.statusCode).to.eql(200);
```

I got a failure while testing this locally against https://awx.ci-ui-20240827.gcp.testing.ansible.com
`'can visit the details page of an instance nested inside an instance group and run health check on it'`
<img width="575" alt="image" src="https://github.com/user-attachments/assets/07103ed3-9ee7-42fb-bae6-4b1e0a7375fc">

Reply:
@vidyanambiar fixed.